### DISCUSSION TITLE
cant able to add github repository to local desktop

### DISCUSSION THREAD

**Discussion by raju-b70** [learndevopsonline/learndevopsonline#1840] [Q&A]
![Screenshot (22)](https://github.com/learndevopsonline/learndevopsonline/assets/161826606/bedc72c2-2794-4ff2-b240-c9927ddd6859)

**Comment by chaithanya1812** [comment]
Before the URL, you need to use 
git clone  <url>

**Reply by raju-b70** [reply]
![Screenshot (23)](https://github.com/learndevopsonline/learndevopsonline/assets/161826606/5392aa14-7f0c-4d00-a2fc-740f96f694bb)

**Reply by raju-b70** [reply]
i tried

**Comment by chaithanya1812** [comment]
Join the lab query session once raju..
#  Same thing I done for me it's working
![image](https://github.com/learndevopsonline/learndevopsonline/assets/111736742/63aa2604-0e38-4b89-adf4-43dffdee3397)

**Reply by raju-b70** [reply]
![Screenshot (24)](https://github.com/learndevopsonline/learndevopsonline/assets/161826606/3ff4f67c-1b39-4257-b4fa-c405b6b85e6e)
thank you i got it when i tried with d:drive . but i just  want to know why i  got same error when i tried to add in c drive

**Comment by chaithanya1812** [comment]
# Issue may be you are in c drive Users directory..
## Check once more time other than User directory, Try with other directory It will Work
